### DISCUSSION TITLE
Constantly get 502 Bad Gateway in Chrome Dev Tools (but it works?)

### DISCUSSION THREAD

**Discussion by DeFiEcoSys** [WalletConnect/walletconnect-monorepo#3003] [Developer Support] [answered]
Hey all,

I've just put WC into my project, I've got it working and it loads the values, but it puts error messages (about v1!) in the console. 

`import { EthereumClient as t, w3mConnectors as e, w3mProvider as n, WagmiCore as a, WagmiCoreChains as i, WagmiCoreConnectors as r } from "https://unpkg.com/@web3modal/ethereum";
import { Web3Modal as s } from "https://unpkg.com/@web3modal/html";
let { configureChains: o, createConfig: l, getAccount: d, prepareWriteContract: c, writeContract: u, readContract: p } = a,
  { bsc: y } = i,
  // { CoinbaseWalletConnector: m } = r,
  chains = [y],
  projectId = "xxx",
  { publicClient: w } = o(chains, [n({ projectId })]),
  wagmiConfig = l({ autoConnect: !0, connectors: e({ projectId, chains }), publicClient: w }),
  ethereumClient = new t(wagmiConfig, chains),
  web3modal = new s({ projectId }, ethereumClient);

async function getTotalInvested() {
  try {
    let t = await p({ address: contractAddress, abi: contractABI, functionName: "getTotalInvested" });
    // console.log(t);
    let e = Number(t.toString()),
      n = formattedNumber(e, 2);
    return (document.getElementById("total-invested").innerHTML = n + " BUSD"), n;
  } catch (a) {
    console.error(a);
  }
}`

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/95844277/b0aec5b7-9c99-4279-8d3f-ef3587401a90)

**Comment by crypblizz8** [comment, marked as answer]
Hey there, is this resolved for you now?

**Reply by DeFiEcoSys** [reply]
Yay resolved. Thanks!
